### ISSUE TITLE
CLI help?

### ISSUE THREAD

**Issue by ondrejhlavacek** [keboola/component-generator#43]
```
Please enter vendor id: keboola
Please enter component id (including vendor id): keboola.processor-filter-files
Please enter service account name: keboola+processor_filter_files_travis
Please enter service account password: ***
```

Možná by tam mohlo bejt `Please enter Keboola Developer Portal vendor id`, nejvíc mě to zmátlo u `Please enter service account name`.

**Comment by tomasfejfar** [comment]
Konec konců by to rovnou mohlo složit component id z vendora a component name. resp. být chytřejší - tj. když to nezačíná na vendor id, tak doplnit.

**Comment by odinuv** [comment]
jo, mohlo. Ja to tam puvodne nedal schvalne, protoze to u nekterych starsich komponent neni (id komponenty neobsahuje id vendora), ale tam se generator stejne nepouziva, takze to neni problem. A ubyde jedna polozka pro zadani.
